man/pam_systemd.xml

```diff
@@ -143,6 +143,10 @@
                 <entry><constant>manager-early</constant></entry>
                 <entry>Similar to <constant>manager</constant>, but for the root user. Compare with the <constant>user</constant> vs. <constant>user-early</constant> situation. (Added in v256.)</entry>
               </row>
+              <row>
+                <entry><constant>none</constant></entry>
+                <entry>Disables logind for this session. No user manager and no session scope will be created (Added in v257.)</entry>
+              </row>
```

Comment:
Can you please register this properly in src/login/logind-session.h in the enum. It's in the same namespace after all, hence it should be listed among the other values. I'd really parse it like the others, but of course refuse it when someone actually tries to register a session of this type with logind.

This doesn't "disable logind", it disbles "registration of this session with systemd-logind".

And I think the 2nd sentence should read: "Not session scope unit will be created, and thus also not user service manager be started."

also, this is v258 material